packages

```diff
@@ -0,0 +1,72 @@
+autoconf
```

Comment:
This is linked to the Containerfile I guess, I think this is not needed to build the project too ?

Also the `justfile` seems to be an oversight ?